src/toil_scripts/adam_gatk_pipeline/align_and_call.py

```diff
@@ -117,132 +117,30 @@
 # import boto sdb connection for autoscaling
 import boto.sdb
 
+import textwrap
```

Comment:
Yeah, it's a really small thing to do, but it really cleans up diffs when someone adds a new import.